src/core/types.ts

```diff
@@ -0,0 +1,111 @@
+export enum EVMNetworkType {
+  ETHEREUM_MAINNET = 'ETHEREUM_MAINNET',
+  ETHEREUM_GOERLI_TESTNET = 'ETHEREUM_GOERLI_TESTNET',
+  POLYGON_MAINNET = 'POLYGON_MAINNET',
+  AVALANCHE_MAINNET = 'AVALANCHE_MAINNET',
+  AVALANCHE_FUJI_TESTNET = 'AVALANCHE_FUJI_TESTNET',
+}
+
+export enum Chain {
+  ETHEREUM_MAINNET = 1,
+  ETHEREUM_GOERLI_TESTNET = 5,
+  POLYGON_MAINNET = 137,
+  AVALANCHE_MAINNET = 43_114,
+  AVALANCHE_FUJI_TESTNET = 43_113,
+}
+
+export type EVMNetworkLike<T extends EVMNetworkType, C extends Chain> = {
+  readonly type: T;
+  readonly chainId: C;
+};
+
+export type Network =
+  // | SolanaNetwork
+  EVMNetworkLike<EVMNetworkType, Chain>;
+
+export enum PaymentToken {
+  SENTINEL = 0, // denotes non-existence of payment token. i.e. default value signifying it hasn't been set
+  WETH = 1,
+  DAI = 2,
+  USDC = 3,
+  USDT = 4,
+  TUSD = 5,
+  RENT = 6 /* reserved for when RENT token is deployed */,
+  ACS = 7,
+  WELT = 8,
+  KNIGHT = 9,
+  TOSHI = 10,
+  ELLE = 11 /* Kryptoria */,
```

Comment:
can we add a TODO here please: `TODO: remove if they don't integrate`. Reason being I have only added this so that they can test on goerli, but they have never integrated with us, so we might as well remove it later after they confirm they aren't integrating.

Reply:
I've removed it as per https://discord.com/channels/842471156413235261/972150826103627846/1116400672737345546